### PR TITLE
Document process for requesting documentation updates

### PR DESCRIPTION
**Pull request by kenzieschmoll**
@sfshaza2 @atsansone @johnpryan

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## DevTools Documentation - How To Guide
+The documenatation for Dart & Flutter DevTools lives on [flutter.dev](https://docs.flutter.dev/development/tools/devtools/overview).
+
+Our docs are written and pushed up the the [flutter/website](https://docs.flutter.dev/development/tools/devtools/overview)
+repo by the Flutter DevRel team. Developers working on DevTools should collaborate with Flutter tech writers by following
+the below worflow.
+
+## How to request documenation updates
```
Comment (CoderDake) [inline DOCUMENTATION.md:8]:
```suggestion
## How to request documentation updates
```

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## DevTools Documentation - How To Guide
+The documenatation for Dart & Flutter DevTools lives on [flutter.dev](https://docs.flutter.dev/development/tools/devtools/overview).
```
Comment (CoderDake) [inline DOCUMENTATION.md:2]:
```suggestion
The documentation for Dart & Flutter DevTools lives on [flutter.dev](https://docs.flutter.dev/development/tools/devtools/overview).
```

File: DOCUMENTATION.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Our docs are written and pushed up the the [flutter/website](https://docs.flutter.dev/development/tools/devtools/overview)
+repo by the Flutter DevRel team. Developers working on DevTools should collaborate with Flutter tech writers by following
+the below worflow.
```
Comment (CoderDake) [inline DOCUMENTATION.md:6]:
```suggestion
the below workflow.
```

**Review by johnpryan — APPROVED** [review]

File: DOCUMENTATION.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## How to request documenation updates
+
+1. Create a [Screencast](chrome://extensions/?id=ccbdlfckiiklnpopmgihhnegpbcgceja) (preferred) or a Google Drive video
+explaining the new feature or functionality changes that require docs updates. If the feature is quick to document, you
+can include a rough draft of the documentation you think should be written, but where possible, we should leave the
+writing to our talented tech writers.
+2. Create a Go Link pointed at your demo video.
```
Comment (CoderDake) [inline DOCUMENTATION.md:14]:
```suggestion
2. [Create a Go Link](https://moma.corp.google.com/go2/) pointed at your demo video.
```

**Review by CoderDake — APPROVED** [review]

File: DOCUMENTATION.md (hunk 1 above)
Comment (polina-c) [inline DOCUMENTATION.md]: This section explains how to document user flows.

Other types of documentation (like glossaries, high level overviews or coding recipes) cannot be illustrated by a video and require tech review first. 

Would it make sense to create another section for them?

Comment (kenzieschmoll) [reply DOCUMENTATION.md]: added another section

**Pull request merged**